proposals/0002-rest-spread-operator.md

```diff
@@ -0,0 +1,133 @@
+# Feature name
```

Comment:
This one is supposed to be changed, I'll make it more obvious in the template :)